### PR TITLE
doc: small fix of "Getting Started" document structure in section docker

### PR DESCRIPTION
**Pull request by gschorcht**
### Contribution description

This PR fixes the following problems:

- Section "Usage" which describes the usage of docker is on heading level 1 but has to be on  heading level 2.
- Section "Setup" has only one subsection. A second subsection is added. Now, the section "Setup" has two subsections, one that describes the installation of docker and one that describes downloading and testing of the RIOT docker build image.
- Backticks in headings no longer seem to work in doxygen markdown support.

screenshot_001

### Testing procedure

`make doc` should give the following results for `doc/doxygen/html/getting-started.html`

screenshot_002

### CODE REVIEW

File: doc/doxygen/src/getting-started.md [hunk 1]
```diff
@@ -321,7 +323,8 @@ This will take a while. If it finishes correctly, you can then use the toolchain
 $ docker run --rm -i -t -u $UID -v $(pwd):/data/riotbuild riot/riotbuild ./dist/tools/compile_test/compile_test.py
 ```
 
-# Usage
+Usage
+-------
```
Comment (aabadie) [inline doc/doxygen/src/getting-started.md:327]:
```suggestion
Usage
-----
```

**Review by aabadie — APPROVED** [review]
I confirm the issue and that it's fixed by this PR. Please squash the small nit above (or just ignore if you prefer).

ACK

File: doc/doxygen/src/getting-started.md (hunk 1 above)
Comment (gschorcht) [reply doc/doxygen/src/getting-started.md:327]: Done

**Comment by gschorcht** [comment]
Thanks for reviewing and merging.

**Pull request merged**